lib/buyers/views.py

```diff
@@ -14,6 +19,13 @@ class BuyerViewSet(NonDeleteModelViewSet):
     serializer_class = BuyerSerializer
     filter_fields = ('uuid', 'active')
 
+    @detail_route(methods=['post'])
```

Comment:
the only issue I have with `detail_route` is you didn't copy over tests which means it would be hard to upgrade if we need an intermediary release before 3. Hopefully we can go straight to 3 though.